### PR TITLE
Update README; Add CHANGELOG

### PR DESCRIPTION
**Pull request by richb-hanover**
Update README to emphasize pre-built binaries;
Editorial pass over the README;
Full description of all command line options;
Troubleshooting section;
New screenshots;
Add CHANGELOG;
~~Bump version to 0.1.0 with "-testing" suffix for development versions;
   Remove that suffix when a release ships~~

### CODE REVIEW

File: src/Cargo.lock [hunk 1]
```diff
@@ -1158,7 +1158,7 @@ dependencies = [
 
 [[package]]
 name = "crusader-lib"
-version = "0.0.12"
+version = "0.1.0-testing"
```
Comment (Zoxc) [inline src/Cargo.lock:1161]: This is outdated.

File: README.md [hunk 2]
```diff
@@ -1,84 +1,202 @@
...
 
-Click the **Result** tab to see a chart similar to the one above.
+* To get the git commit hash of the current checkout,
+  use `git rev-parse --short master`
 
-<img src="media/gui.png">
+* I sometimes see this message: 
+`Warning: Load termination timed out. There may be residual untracked traffic in the background.` 
+  
+  Should I be concerned? No. That may happen due to the TCP termination being lost or TCP incompatibilities between OSes. It's likely benign if you see bandwidth and latency drop to idle values after the tests in the graph.
```
Comment (Zoxc) [inline README.md:202]: `bandwidth` -> `throughput`

**Comment by richb-hanover** [comment]
Good catch. I will fix them later tonight.

File: src/Cargo.lock (hunk 1 above)
Comment (richb-hanover) [reply src/Cargo.lock]: fixed

File: README.md (hunk 2 above)
Comment (richb-hanover) [reply README.md]: fixed

**Comment by richb-hanover** [comment]
@Zoxc - I fixed both these. Thanks for pointing them out.

**Comment by richb-hanover** [comment]
Thanks!

**Pull request merged**